Makefile.dep

```diff
@@ -15,6 +15,9 @@
 include $(RIOTBASE)/sys/Makefile.dep
 include $(RIOTBASE)/drivers/Makefile.dep
 
+# pull Makefile.dep of each driver modules if they exist
+-include $(USEMODULE:%=$(RIOTBASE)/drivers/%/Makefile.dep)
```

Comment:
```suggestion
-include $(sort $(USEMODULE:%=$(RIOTBASE)/drivers/%/Makefile.dep))
```

This will fix having the list included multiple times.

Reply:
Addressed